js/bloodmallet_standard.js

```diff
@@ -600,6 +601,11 @@ document.addEventListener("DOMContentLoaded", function () {
     $WowheadPower.refreshLinks();
   });
 
+  document.getElementById("relative_azerite_forge").addEventListener("change", function (e) {
+    relative_azerite_string = e.target.checked;
+    set_relative_azerite_cookie()
```

Comment:
While not necessary, I appreciate `;` at the end of each instruction. Please add them. I'll mark the others, if I see them.